_posts/0000-01-02-elisaep.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Keep your enemies close and your friends closer just in case they turn out to be bastards.